src/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.Specific.cs

```diff
@@ -503,6 +504,47 @@ public void Windows_SetReadModeTo__PipeTransmissionModeByte()
             }
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        public void NamedPipeServer_Connects_With_UnixDomainSocketEndPointClient()
+        {
+            string pipeName = Path.Combine("/tmp", "CoreFxTests_" + Path.GetRandomFileName());
```

Comment:
Why use the same path here as is used when we create a name?  What I was thinking is that we should just pick an arbitrary path and make sure it works, like someone would be doing on their own.  The CoreFxTests_ prefix is meant to be an implementation detail.

Reply:
Arbitrary path: ok let me try $HOME (rooted of course). The prefix is not matching the internal one it is just to help identify the source if it leaks, I will change to something *really* different.

Reply:
> The prefix is not matching

Oh, ok, then ignore my comment.

Reply:
If you thought that, it is likely that other will think the same, changed to avoid any confusion.